code/src/components/Album.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Icons from './Icons';
+
+const Album = (props) => {
+  const { albumsArray } = props
+  return (
+    <section className="album-container">
+      {albumsArray.map((mappedOutItem) => (
+        <div className="album-card" key={mappedOutItem.id}>
+          <div className="image-container">
+            <img src={mappedOutItem.images[0].url} className="album-image" alt="Cover" />
+            <Icons />
+          </div>
+          <div className="albums">
+            <a href={mappedOutItem.external_urls.spotify} target="_blank" className="album-name" rel="noreferrer"> {mappedOutItem.name}</a>
```

Comment:
The `mappedOutItem` is a bit confusing variable name for the album object, it might be better to use `album` or `albumObject`.

Reply:
I agree, it makes the code a bit more convoluted. Other than that it's very easy to read.